python/datafusion/dataframe.py

```diff
@@ -223,6 +223,30 @@ def limit(self, count: int, offset: int = 0) -> DataFrame:
         """
         return DataFrame(self.df.limit(count, offset))
 
+    def head(self, n: int) -> DataFrame:
```

Comment:
Would it be helpful to have a default `n`?

Reply:
Yes, lets go n=5